haproxy/templates/deployment.yaml

```
@@ -40,6 +40,10 @@ spec:
             - name: metrics
               containerPort: 9000
             {{- end }}
+            {{- if .Values.redisk8s.metrics.enabled or .Values.galerak8s.metrics.enabled}}
```

I'm not sure this works. Have you tested it with `helm template -f mytestvaluesfile.yaml` where one of the conditions is `true`?
```suggestion
            {{- if or .Values.redisk8s.metrics.enabled .Values.galerak8s.metrics.enabled}}
```

---

Correct. And I found another typo in there. Fixed both